scene/add-3d-tiles-layer/hs_err_pid25752.log

```diff
@@ -0,0 +1,972 @@
+#
```

Comment:
This file should be deleted. Might be worth adding a git ignore for .log too..

Reply:
Ooops, good catch - thank you for pointing out!